app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    collected_data = {}
+
+    def wrapper(*args) -> int:
```

Comment:
We should provide type annotation `Any` in this case. Your decorator might return different types(int, float and etc.)